lib/matplotlib/colors.py

```diff
@@ -1215,6 +1247,18 @@ class ListedColormap(Colormap):
             N > len(colors)
 
         the list will be extended by repetition.
+
+        .. deprecated:: 3.11
```

Comment:
This seems orthogonal to this PR?  Or is this already deprecated elsewhere and this is just a doc cleanup?  I'm not really following why we would need to do this.

Reply:
The deprecation is in #29135, it's indeed unrelated. I just noticed that I haven't added this to the parameter docs, and didn't bother to make a separate PR since I was editing that position in the docstring anyway. Sorry for the confusion.